ui/raidboss/data/06-ew/dungeon/ktisis_hyperboreia.txt

```diff
@@ -0,0 +1,172 @@
+### KTISIS HYPERBORAEIA
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 00:0839::.*is no longer sealed/ window 100000 jump 0
+
+#~~~~~~~#
+# LYSSA #
+#~~~~~~~#
+
+# -ii 1961 6E90 -it Lyssa
+
+0.0 "--sync--" sync / 00:0839::The Frozen Sphere will be sealed off/ window 0,1
+10.7 "Skull Dasher" sync / 1[56]:[^:]*:Lyssa:625E:/ window 10.7,10
+21.2 "Frostbite and Seek" sync / 1[56]:[^:]*:Lyssa:6257:/
+23.2 "--untargetable--"
+34.3 "Punishing Slice #1" sync / 1[56]:[^:]*:Lyssa:6258:/ window 34.3,30
+35.2 "Punishing Slice #2" sync / 1[56]:[^:]*:Lyssa:6259:/
```

Comment:
I think 6258 is a self-targeted ability and 6259 is the damage.  I'd say leave 6258 and ignore 6259?

I'm also not sure why this one has a window.  Is there a variable time here?

Reply:
Oops. Thanks for the catch. It's a weird implementation:

```
20|2021-12-05T18:01:43.3250000-08:00|4007ADDB|Lyssa|6259|Punishing Slice|4007ADDB|Lyssa|1.70|-144.00|36.50|496.00|0.00|fdff86979b2d5c95
21|2021-12-05T18:01:44.3500000-08:00|4007ADD8|Lyssa|6258|Punishing Slice|4007ADD8|Lyssa|1B|62588000|0|0|0|0|0|0|0|0|0|0|0|0|0|0|1697481|1830080|10000|10000|||-144.03|49.00|496.00|0.00|1697481|1830080|10000|10000|||-144.03|49.00|496.00|0.00|00057627|0|f25b9801b3f54a12
21|2021-12-05T18:01:45.2830000-08:00|4007ADDB|Lyssa|6259|Punishing Slice|E0000000||0|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0|||||||||||44|44|0|10000|||-144.00|36.50|496.00|0.00|00057628|0|04473ca37e07f5e1
```

Are you sure we want to sync to 6258? That would make the damage visual sync about a second after the timeline and that might be confusing for people. (6258 is instant and invisible, 6259 has a short cast time.)

The window is simply because I put wide windows everywhere we have isolated abilities as a failsafe. (If the timeline fails to start on the initial lines, if the user opens ACT late, etc. It costs us nothing to have the most forgiving syncs possible, and it's a much nicer user experience if the timeline can "catch up" after forgetting ACT.)

Reply:
Ah, I see.  Maybe 6259 makes more sense then here!